code/game/machinery/doors/airlock.dm

```diff
@@ -1604,6 +1605,30 @@
 	else if(istype(note, /obj/item/photo))
 		return "photo"
 
+/obj/machinery/door/airlock/AltClick(mob/living/user)
+	if(!istype(user))
+		user << "<span class='info'>Nice try, ghosts.</span>"
```

Comment:
to_chat

Reply:
KOR!!! how could you commit such a sin?!?!